src/components/FilesGallery/__stories__/FilesGallery.stories.tsx

```diff
@@ -0,0 +1,172 @@
+import React from 'react';
+
+import {ArrowDownToLine, Link} from '@gravity-ui/icons';
+import {
+    ActionTooltip,
+    Button,
+    CopyToClipboard,
+    Icon,
+    Text,
+    usePortalContainer,
+} from '@gravity-ui/uikit';
+import type {Meta, StoryFn} from '@storybook/react';
+
+import {FilesGallery, FilesGalleryProps} from '../FilesGallery';
+import {DocumentFileView} from '../components/DocumentFileView/DocumentFileView';
+import {ImageFileView} from '../components/ImageFileView/ImageFileView';
+import {VideoFileView} from '../components/VideoFileView/VideoFileView';
+import {GalleryFileBase} from '../types';
+
+export default {
+    title: 'Components/FilesGallery',
+    component: FilesGallery,
+    parameters: {
+        a11y: {
+            element: '#storybook-root',
+            config: {
+                rules: [
+                    {
+                        id: 'color-contrast',
+                        enabled: false,
+                    },
+                ],
+            },
+        },
+    },
+} as Meta<typeof FilesGallery>;
+
+type GalleryFile = GalleryFileBase &
+    (
+        | {
+              type: 'image' | 'video' | 'document';
+              url: string;
+          }
+        | {type: 'text'; text: string}
+    );
+
+const files: GalleryFile[] = [
+    {
+        type: 'image',
+        url: 'https://santreyd.ru/upload/iblock/acc/accd0c751590e792f7e43a05f22472f9.jpg',
+        imageSrc: 'https://santreyd.ru/upload/iblock/acc/accd0c751590e792f7e43a05f22472f9.jpg',
+        data: {
+            name: 'Corgi image',
+            type: 'image/jpeg',
+        } as File,
+    },
+    {
+        type: 'video',
+        url: 'http://commondatastorage.googleapis.com/gtv-videos-bucket/sample/BigBuckBunny.mp4',
+        data: {
+            name: 'Bunny Film',
+            type: 'video/mp4',
+        } as File,
+    },
+    {
+        type: 'text',
+        text: 'Lorem Ipsum - это текст-"рыба", часто используемый в печати и вэб-дизайне. Lorem Ipsum является стандартной "рыбой" для текстов на латинице с начала XVI века. В то время некий безымянный печатник создал большую коллекцию размеров и форм шрифтов, используя Lorem Ipsum для распечатки образцов. Lorem Ipsum не только успешно пережил без заметных изменений пять веков, но и перешагнул в электронный дизайн. Его популяризации в новое время послужили публикация листов Letraset с образцами Lorem Ipsum в 60-х годах и, в более недавнее время, программы электронной вёрстки типа Aldus PageMaker, в шаблонах которых используется Lorem Ipsum',
+        data: {
+            name: 'Some text',
+            type: 'text',
+        } as File,
+    },
+    {
+        type: 'document',
```

Comment:
![image](https://github.com/user-attachments/assets/2ccf0808-98ae-4c3d-8896-56991ebe2503)
It is not displayed. Is this how it should be?

Reply:
This is just an example, mock pdf file, I can't find a correct link. The service should implement the document functionality itself.